proto/message_api/v1/message_api.proto

```diff
@@ -67,7 +67,11 @@ message PagingInfo {
 
 // Envelope encapsulates a message while in transit.
 message Envelope {
+    // The topic the message belongs to,
+    // MUST be the same as the topic embedded in the message.
     string content_topic = 1;
+    // Message timestamp
+    // MUST be the same as the timestamp embedded in the message.
```

Comment:
MUST might be too strong here. That's only true for the newer types. We do still have other things that fit into the envelope that do not have conforming timestamps

Reply:
👍 

I guess the confusing part is that a lot of the old message types do have timestamps. They just are in milliseconds

Reply:
OK, I can clarify "the same" as "equivalent" maybe? Is that good enough?

Reply:
Works for me

Reply:
I'm also going to fix the issue that causes them to be slightly different for existing messages

Reply:
> I'm also going to fix the issue that causes them to be slightly different for existing messages

I just started looking into that.

Reply:
You can have that one